spec/controllers/decidim/half_signup/quick_auth_controller_spec.rb

```diff
@@ -34,6 +35,10 @@
     request.env["decidim.current_organization"] = organization
     request.env["devise.mapping"] = ::Devise.mappings[:user]
     request.session[:auth_attempt] = auth_session
+
+    Decidim.configure do |config|
```

Comment:
Prefer to define mock for `Decidim.configure.sms_gateway_service` 

Example : `allow(Decidim.config.sms_gateway_service).to receive(:present?).and_return(true)`